packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -134,6 +135,16 @@ class ScaffoldPrelayoutGeometry {
   /// will be 0.0.
   final EdgeInsets minInsets;
 
+  /// The minimum padding to inset interactive elements to be within a safe,
+  /// un-obscured space.
+  ///
+  /// This value is the result of calling [MediaQuery.viewPadding] in the
+  /// [Scaffold]'s [BuildContext], this helps distinguish different types of
+  /// obstructions. For example, by using the viewPadding property, padding
+  /// would defer to the iPhone "safe area" regardless of whether a keyboard is
```

Comment:
We want to make it clear that minViewPadding doesn't change when the soft keyboard goes up and down.

Reply:
I'll add some more clarification. :)